dashboard/config/marketing/announcements.json

```diff
@@ -1,6 +1,6 @@
 {
   "pages": {
-    "/home": "user-research-aug-2024",
+    "/home": "transformers-sept-2024",
     "/student-home": ""
```

Comment:
100% a product question but do we not want to show this for students?

Reply:
Should be for both teachers and students.

Reply:
Added to the student page!